fedimintd/src/ui.rs

```diff
@@ -214,10 +214,21 @@ async fn post_guardians(
 
 #[derive(Template)]
 #[template(path = "params.html")]
-struct UrlConnection {}
+struct UrlConnection {
+    ro_bitcoind_rpc: String,
```

Comment:
Can't this be an `Url`? Iirc that implements `Display`.

Reply:
We don't always have `Url`, and it's `Option<Url>` and then askama's template seems too simplistic to work with it... 

Maybe there's a better way.

Reply:
You could have a `match` statement in askama iirc, but I agree that's probably not much better at that point (although you could hide other HTML elements like labels and headings depending on it).